components/shared/Banner.js

```diff
@@ -0,0 +1,25 @@
+/* @author Charles Liu */
+
+import Image from 'next/image'
+
+import styles from './Banner.module.css'
+
```

Comment:
can you add a function comment right here?

Reply:
will do!